src/main/java/lotto/domain/LottoGroup.java

```diff
@@ -10,7 +14,33 @@ public LottoGroup(List<Lotto> lottoGroup) {
         this.lottoGroup = lottoGroup;
     }
 
+    public LottoGroup(Lotto lotto) {
+        this.lottoGroup = Collections.singletonList(lotto);
+    }
+
     public List<Lotto> getLottoGroup() {
-        return lottoGroup;
+        return Collections.unmodifiableList(lottoGroup);
+    }
+
+    public LottoGroup merge(LottoGroup target) {
+        return new LottoGroup(Stream.concat(lottoGroup.stream(), target.getLottoGroup().stream())
+            .collect(Collectors.toList()));
+    }
```

Comment:
```suggestion
    public LottoGroup merge(LottoGroup target) {
        return new LottoGroup(Stream.concat(lottoGroup.stream(), target.lottoGroup.stream())
            .collect(Collectors.toList()));
    }
```
getter를 사용하지 않고 이렇게도 가능하겠네요!